pkg/config/config.go

```diff
@@ -1361,6 +1361,10 @@ func load(config Config, origin string, loadSecret bool) (*Warnings, error) {
 	}()
 
 	if err := config.ReadInConfig(); err != nil {
+		if IsServerless() {
+			log.Debugf("Running in serverless environment - config file is expectedly missing: %v", err)
```

Comment:
Maybe just `No config file detected, using environment variable based configuration`. We don't need to tell customers they are running their code in a serverless environment, it should be obvious based on the source.

Reply:
Ahh, I just saw @sgnn7 's comment. Sorry for the conflicting feedback.

Reply:
All good 👍 makes sense and it's clear and concise